examples/sql/sql_example.py

```diff
@@ -0,0 +1,99 @@
+import hazelcast
+from hazelcast.serialization.api import Portable
+
+from hazelcast.sql import SqlStatement
+
+
+class Customer(Portable):
+    def __init__(self, name=None, age=None, is_active=None):
+        self.name = name
+        self.age = age
+        self.is_active = is_active
+
+    def write_portable(self, writer):
+        writer.write_string("name", self.name)
+        writer.write_int("age", self.age)
+        writer.write_boolean("is_active", self.is_active)
+
+    def read_portable(self, reader):
+        self.name = reader.read_string("name")
+        self.age = reader.read_int("age")
+        self.is_active = reader.read_boolean("is_active")
+
+    def get_factory_id(self):
+        return 1
+
+    def get_class_id(self):
+        return 1
+
+
+client = hazelcast.HazelcastClient(portable_factories={1: {1: Customer}})
+
+customers = client.get_map("customers").blocking()
+
+# Fill the customers map with some values
+customers.set(1, Customer("Peter", 42, True))
+customers.set(2, Customer("John", 23, False))
+customers.set(3, Customer("Joe", 33, True))
+
+# Project a single column that fits the criterion
+result = client.sql.execute("SELECT name FROM customers WHERE age < 35 AND is_active")
+
+for row in result:
+    # Get the object with the given column name in the row
+    name = row.get_object("name")
+    print(name)
+
+
+# One can also select all fields
```

Comment:
The sample is actually just below it. I just used one statement to show two different things
`SELECT * FROM customers`

Reply:
OK, I think removing the blank line after comment `One can also select all fields`  and the blank line between the next comment and the `with` statement below, it may make it easier to relate the comment with the `with` statement.